src/pages/index.js

```diff
@@ -42,24 +132,50 @@ editFormValidator.enableValidation();
 const addFormValidator = new FormValidator(config, addNewCardFormEl);
 addFormValidator.enableValidation();
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (initialCardData) => {
-      const card = createCard(initialCardData);
-      cardSection.prependItem(card);
-    },
-  },
-  ".cards__list"
+const editAvatarFormValidator = new FormValidator(
+  config,
+  profileEditAvatarFormEl
 );
-
-cardSection.renderItems();
+editAvatarFormValidator.enableValidation();
 
 const addNewCardForm = new PopupWithForm("#add-image-popup", (inputValues) => {
-  const cardEl = createCard(inputValues);
-  cardSection.prependItem(cardEl);
+  addNewCardForm.setLoadingState(true);
```

Comment:
Needs correction: I don't see the loading text on the buttons (profile form, avatar form, card form)

Reply:
fixed